src/pages/SignOut/SignOut.jsx

```diff
@@ -0,0 +1,39 @@
+import { supabase } from '../../services/supabaseClient';
+import React  from 'react';
+
+import { useEffect } from 'react'
+import { useNavigate } from 'react-router';
+import { paths } from '../../services/path';
+
+export default function Logout() {
+  const navigate = useNavigate();
+   useEffect(()=>{
+       const initialize = async () => await supabase.auth.signOut()
+       initialize()
+       navigate(paths.signin)
```

Comment:
This doesn't actually wait for the user to be signed out before redirecting the users away.

the navigate would need to be inside the async function where `signOut` is being `await`ed.

```javascript
const signOut = async () => {
  await supabase.auth.signOut()
  navigate(paths.signin)
}
signOut()
```

or you can use the promise syntax
```javascript
supabase.auth.signOut().then(() => navigate(paths.signin))
```

Reply:
Thanks.